ci-operator/step-registry/cucushift/upgrade/healthcheck/cucushift-upgrade-healthcheck-commands.sh

```diff
@@ -205,8 +220,7 @@ OC="run_command_oc"
 
 #1. Make sure all machines are applied with latest machineconfig
 echo "Step #1: Make sure all machines are applied with latest machineconfig"
```

Comment:
how about update the description here accordingly?

Reply:
done.